sdcm/chaos_mesh.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+
+from sdcm.cluster_k8s import KubernetesCluster
+from sdcm.utils.k8s import HelmValues, KubernetesOps
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ChaosMesh:  # pylint: disable=too-few-public-methods
+    NAMESPACE = "chaos-mesh"
+    VERSION = "2.5.0"
+    HELM_SETTINGS = {'dashboard.create': True, 'dnsServer.create': True}
+
+    def __init__(self):
+        self._k8s_cluster = None
+
+    def initialize(self, k8s_cluster: KubernetesCluster) -> None:
+        """Installs chaos-mesh on k8s cluster and prepares for future k8s chaos testing."""
+        if self._k8s_cluster is not None:
+            LOGGER.warning("Chaos Mesh is already initialized. It should have not been initialized twice.")
+            return
+        self._k8s_cluster = k8s_cluster
+        self._install()
+
+    def _install(self):
+        if self._k8s_cluster.kubectl("kubectl get ns chaos-mesh", ignore_status=True).ok:
+            LOGGER.info("Chaos Mesh is already installed. Skipping installation.")
+            return
+        LOGGER.info("Installing chaos-mesh...")
+        self._k8s_cluster.helm("repo add chaos-mesh https://charts.chaos-mesh.org")
+        self._k8s_cluster.helm('repo update')
+        self._k8s_cluster.kubectl("create namespace chaos-mesh")
+        self._k8s_cluster.helm_install(
+            target_chart_name="chaos-mesh",
+            source_chart_name="chaos-mesh/chaos-mesh",
+            version=self.VERSION,
+            use_devel=False,
+            namespace=self.NAMESPACE,
+            values=HelmValues(self.HELM_SETTINGS),
+            atomic=True,
+            timeout="20m"
+        )
+        self._wait_for_chaos_mesh_readiness(timeout=20)
```

Comment:
the time out is in seconds ? or minutes ?, bare timeout across the code is in seconds. 
while `readiness_timeout` used inside the function is in minutes.

Reply:
I followed pattern from readiness function - it's in minutes. I'll rename it to be clear.